syntax = "proto3";
package permission;




import "google/protobuf/struct.proto";

/*
SavePermission返回
*/
message SavePermissionResponse {
message Data {
    /*
    权限点id
    */
    string id = 1 ;
}
        
    
    /*
    返回码
    */
    int32 code = 1 ;
    /*
    结果信息
    */
    string msg = 2 ;
    /*
    数据
    */
    Data data = 3 ;
}

/*
SavePermissionApi返回
*/
message SavePermissionResponseWrapper {
    /*
    返回码
    */
    int32 code = 1 ;
    /*
    返回码解释
    */
    string codeExplain = 2 ;
    /*
    错误详情
    */
    string error = 3 ;
    /*
    返回数据
    */
    permission.SavePermissionResponse data = 4 ;
}

